avatar/avatar.html

```diff
@@ -0,0 +1,65 @@
+            </ul>
+        </header>
+    </nav>
+    <section class="middleSection">
+        <div class="siderbar">
+            <ul class=" d-flex align-center">
+                <li class="sliderbarItem flex-1 active_item"><a href="./avatar.html">Avatar</a></li>
+                <li class="sliderbarItem flex-1"><a href="../images/images.html">Images</a></li>
+                <li class="sliderbarItem flex-1"><a href="">Avatar</a></li>
+                <li class="sliderbarItem flex-1"><a href="">Avatar</a></li>
+                <li class="sliderbarItem flex-1"><a href="">Avatar</a></li>
+                <li class="sliderbarItem flex-1"><a href="">Avatar</a></li>
+                <li class="sliderbarItem flex-1"><a href="">Avatar</a></li>
+                <li class="sliderbarItem flex-1"><a href="">Avatar</a></li>
+            </ul>
+        </div>
+        <div class="contentBox">
+            <div class="d-flex align-center justify-around w-half">
+                <div class="avatar sm">
+                    <img src="./avatar-img.jpg" alt="avatar-img">
+                </div>
+                <div class="avatar md">
+                    <img src="./avatar-img.jpg" alt="avatar-img">
+                </div>
+                <div class="avatar lg">
+                    <img src="./avatar-img.jpg" alt="avatar-img">
+                </div>
+            </div>
+            <div class="doc-img">
+                <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2509%2509%2509%2520%253Cdiv%2520class%253D%2522avatar%2520sm%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522.%252Favatar-img.jpg%2522%2520alt%253D%2522avatar-img%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar%2520md%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522.%252Favatar-img.jpg%2522%2520alt%253D%2522avatar-img%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar%2520lg%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522.%252Favatar-img.jpg%2522%2520alt%253D%2522avatar-img%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E"
+                style="width: 698px; height: 354px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
try to create separate style for iframe in CSS section

Reply:
Okay i will do it